package.json

```diff
@@ -35,6 +35,7 @@
     "gulp-babel": "6.1.2",
     "gulp-eslint": "2.0.0",
     "gulp-uglify": "1.5.3",
-    "mysql": "2.10.2"
+    "mysql": "2.10.2",
+    "ora": "^0.2.1"
```

Comment:
We should lock the version of `ora` at `"0.2.1"` in case future updates break our build.